src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.html

```diff
@@ -23,26 +23,30 @@ <h1 class='slds-text-body_regular' data-qa-locator='heading'>{labels.elevateWidg
                                             </lightning-button>
                                         </template>
                                         <template if:false={isDisabled}>
-                                            <lightning-button variant="base"
-                                                label={labels.cancelButtonLabel}
-                                                icon-name='utility:close'
-                                                icon-position="right"
-                                                onclick={handleUserDisabledWidget}
-                                                class='slds-button slds-button_base slds-float_right hide-card-button'
-                                                data-qa-locator='button Cancel Update Payment Information'>
-                                            </lightning-button>
+                                            <template if:false={updatePaymentMode}>
+                                                <lightning-button variant="base"
+                                                    label={labels.cancelButtonLabel}
+                                                    icon-name='utility:close'
+                                                    icon-position="right"
+                                                    onclick={handleUserCancelledWidget}
+                                                    class='slds-button slds-button_base slds-float_right hide-card-button'
+                                                    data-qa-locator='button Cancel Update Payment Information'>
+                                                </lightning-button>
+                                            </template>
                                         </template>
                                     </template>
-                                    <template if:false={isEditPayment}>
+                                    <template if:false={isEditMode}>
                                         <template if:false={isDisabled}>
```

Comment:
Is it possible for the form to be in edit mode and be disabled at the same time? Was wondering if we still needed this isDisabled check here.

Reply:
Yep! This happens when you click Edit on a non-elevate RD2 record that is a "Credit Card" or "ACH" payment, the widget loads in Edit mode, but in the Disabled state where you can click "alternatively, enter payment details now".